Server/rainbow-server/Sources/Application/Model/EntryPersistence.swift

```diff
@@ -42,6 +42,29 @@ extension ScoreEntry {
             }
         }
         
+        // Update document        
+        static func update(id: String, entry: ScoreEntry, to client: CouchDBClient, completion: @escaping (_ entryID: String?, _ error: Error?) -> Void) {
+            getDatabase(from: client) { database, error in
+                guard let database = database else {
+                    return completion(nil, error)
+                }
+                var entryCopy = entry
+                guard let updatedCopy = entryCopy.toJSONDocument() else {
+                    return completion(nil, RainbowPersistenceError.noAvatar)
```

Comment:
I probably need to re-write this function to throw this specific error, because at first it seemed out of the blue here, but I see why you used it 😄